src/components/hooks/useForm/useForm.ts

```diff
@@ -16,8 +14,8 @@ export interface IFieldProps {
   error: string | undefined;
   touched: boolean;
   value: TValue;
-  onChange: (e: any) => void;
-  onBlur: () => void;
+  onChange: (eventData: unknown) => void;
+  onBlur: (eventData: unknown) => void;
```

Comment:
maybe a better type would be:
```
  onChange: React.ChangeEventHandler<Element>;
  onBlur: React.FocusEventHandler<Element>;
```

Reply:
It would, however in this implementation `onChange` and `onBlur` not always expect the actual event object, sometimes [they receive the value directly](https://github.com/zopaUK/react-components/blob/master/src/components/organisms/Form/FormRadioField/FormRadioField.tsx#L14) ( cc. @thegrinder  ) hence I typed it as `unknown`.

I'll try to refactor the `<Form.* />` components so that they all send the actual event to this handlers 💆🏻‍♂️